### ISSUE TITLE
java.lang.RuntimeException: Could not load storage!

### ISSUE THREAD

**Issue by jmbillingsley** [erhard-lab/gedi#51]
Greetings!
I'm attempting to run GRAND SLAM on a high performance cluster. I believe it installed correctly and the indexed reference was created correctly.  When I run it, i get the following error:
2020-07-29 15:34:36.150 INFO OS: Linux 3.10.0-1062.el7.x86_64 amd64
2020-07-29 15:34:36.402 INFO Java: Java HotSpot(TM) 64-Bit Server VM 1.8.0_112-b15
2020-07-29 15:34:36.534 INFO Discovering classes in classpath
2020-07-29 15:34:36.680 INFO Preparing simple class references
2020-07-29 15:34:36.794 INFO Gedi version 1.0.2 (JAR) startup
2020-07-29 15:34:36.796 INFO Command: gedi -e Slam -genomic homo_sapiens.100 -progress -plot -D -prefix slamout -reads newbamfilelist.txt
2020-07-29 15:34:36.797 INFO GRAND-SLAM version 2.0.5f
2020-07-29 15:34:36.957 INFO Reading oml /home/jmb17/.gedi/genomic/homo_sapiens.100.oml
2020-07-29 15:34:36.963 INFO Done reading oml /home/jmb17/.gedi/genomic/homo_sapiens.100.oml
Slam
Slam is an analysis method for SLAM/scSLAM-seq data.

Error:
Could not load storage!

gedi -e Slam <Options>

General:
 -prefix <prefix>      The prefix used for all output files
 -genomic <genomic>    The indexed GEDI genome.
 -reads <reads>        The mapped reads from the SLAM-seq experiment.

Commandline:
 -progress             Show progress
 -D                    Verbose output of errors
 -h                    Show usage
 -hh                   Show verbose usage
 -hhh                  Show extra verbose usage
 -dry                  Dry run
 -keep                 Do not remove temp files

java.lang.RuntimeException: Could not load storage!
	at gedi.core.region.GenomicRegionStorage.load(GenomicRegionStorage.java:458)
	at gedi.util.program.parametertypes.StorageParameterType.parse(StorageParameterType.java:33)
	at gedi.util.program.parametertypes.StorageParameterType.parse(StorageParameterType.java:29)
	at gedi.util.program.CommandLineHandler.parse(CommandLineHandler.java:104)
	at gedi.util.program.CommandLineHandler.parse(CommandLineHandler.java:58)
	at gedi.util.program.GediProgram.run(GediProgram.java:186)
	at executables.Slam.main(Slam.java:46)
Caused by: java.lang.NullPointerException
	at gedi.core.region.GenomicRegionStorage.load(GenomicRegionStorage.java:456)
	... 6 more
2020-07-29 15:34:38.288 INFO Finished: gedi -e Slam -genomic homo_sapiens.100 -progress -plot -D -prefix slamout -reads newbamfilelist.txt
2020-07-29 15:34:38.290 INFO Took 2s 153ms

Thanks for your thoughts!
James

**Comment by florianerhard** [comment]
Dear James,

the file has to have the extension `.bamlist`, not `.txt`. Sorry, this was not clear, I updated the text in the wiki.

Also, I recommend to convert the files using bamlist2cit first (see [here](https://github.com/erhard-lab/gedi/wiki/GRAND-SLAM#run-grand-slam)).

Best,
Florian

**Comment by jmbillingsley** [comment]
Thanks so much Florian, things are working now. The other thing that I did incorrectly is that I did not index the bam files prior to running the scripts. I think we need bam.bai files (samtools index) in the bam file directory for your scripts to run. I don't think I see this mentioned in the tutorial, but I may have overlooked it, or perhaps it is assumed.

Thanks again!
James

On Jul 29, 2020, at 6:37 PM, florianerhard <notifications@github.com<mailto:notifications@github.com>> wrote:



Dear James,

the file has to have the extension .bamlist, not .txt. Sorry, this was not clear, I updated the text in the wiki.

Also, I recommend to convert the files using bamlist2cit first (see here<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_erhard-2Dlab_gedi_wiki_GRAND-2DSLAM-23run-2Dgrand-2Dslam&d=DwMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=48Km-oOLuR85cc0O65cGA36_skqz_Rn2xJWCRadeFlY&m=UHoL63S2EGOTjxBC-Gi0MRceheRrG5r-uE19z9bDxzo&s=mpY3beseN97H41qbssAfnUS5ztjngk6bGhnJdhktgL8&e=>).

Best,
Florian
